### PR TITLE
Improve base container

### PR DESCRIPTION
**Pull request by doosuu**
## Describe your changes

Improve and fix app build by installing CLI into the base container.

## Checklist - Manual tasks

* [x] Examples are executing successfully
* [x] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [x] Created/updated integration tests.
* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    runs-on: ubuntu-latest
+    name: Wait for container images
+
+    steps:
+      - name: Wait for build-base-images workflow to succeed
+        uses: fountainhead/action-wait-for-check@v1.1.0
+        with:
+          checkName: Building image
+          token: ${{ secrets.GITHUB_TOKEN }}
+          timeoutSeconds: 5400
```
Comment (MP91) [inline .github/workflows/release.yml:35]: is this enough timeout?

File: .github/workflows/sdk-build-container.yml [hunk 2]
```diff
@@ -12,25 +12,24 @@
...
+    branches:
+      - main
 
 jobs:
   build-container:
+    name: Building image
     runs-on: ubuntu-latest
-    #strategy:
-    #  matrix:
-    #    platform: [amd64, aarch64]
```
Comment (MP91) [inline .github/workflows/sdk-build-container.yml:33]: should we still keep this? we will hopefully find a fix soon...

File: .github/workflows/sdk-build-container.yml [hunk 3]
```diff
@@ -49,26 +48,48 @@ jobs:
...
+          if [ ${{ github.event_name }} = "push" ]; then
+            echo "type=registry" >> $GITHUB_OUTPUT
+          else
+            echo "type=oci,dest=./container.tar" >> $GITHUB_OUTPUT
+          fi
 
-      - name: "Build image"
+      - name: Build image
         id: image_build
         uses: docker/build-push-action@v3.2.0
```
Comment (MP91) [inline .github/workflows/sdk-build-container.yml:87]: any reason to not use v5?

File: .github/workflows/release.yml (hunk 1 above)
Comment (doosuu) [reply .github/workflows/release.yml:35]: Good catch, increased timeout.

File: .github/workflows/sdk-build-container.yml (hunk 3 above)
Comment (doosuu) [reply .github/workflows/sdk-build-container.yml:87]: No. Just a copy & paste issue. Fixed.

File: .github/workflows/sdk-build-container.yml (hunk 2 above)
Comment (doosuu) [reply .github/workflows/sdk-build-container.yml:33]: Will hopefully be a multiarch build in case it is fixed, not a strategy.

**Review by MP91 — APPROVED** [review]
lgtm

File: .github/workflows/sdk-build-container.yml [hunk 4]
```diff
@@ -12,25 +12,24 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-name: SDK - Build Base Container
+name: SDK - App Builder
```
Comment (MP91) [inline .github/workflows/sdk-build-container.yml:15]: Maybe App Builder is a bit misleading, maybe something with App-Baseimage?

**Review by MP91 — CHANGES_REQUESTED** [review]

File: .github/workflows/sdk-build-container.yml (hunk 4 above)
Comment (doosuu) [reply .github/workflows/sdk-build-container.yml:15]: It is only used as "builder image" for the build stage of app images which I wanted to visualize here. "from ...app_builder as builder"

Comment (MP91) [reply .github/workflows/sdk-build-container.yml:15]:
```suggestion
name: Create App Builder image
```
?

**Review by MP91 — APPROVED** [review]
lgtm

**Pull request merged**